src/vector/index.ts

```diff
@@ -130,6 +128,15 @@ async function start() {
         await settled(rageshakePromise);
 
         const fragparts = parseQsFromFragment(window.location);
+        //:tchap: get the indexedDB store version
+        let indexedDB;
+        try {
+            indexedDB = window.indexedDB;
+        } catch (e) {}
+
+        const needRefresh = await TchapClientUtils.doesNeedRefresh(indexedDB);
```

Comment:
This raises an error if `indexedDB` is not set. It will be catched and leads to `Your Element is misconfigured`. Don't know if this is the desired behaviour.

Reply:
I've taken the same logic from https://github.com/matrix-org/matrix-react-sdk/blob/f7159b859c9e64d8591c66ea72cf092c7eecb300/src/utils/createMatrixClient.ts#L34

Reply:
fixed in latest commits, thanks !